packages/api/src/parse-cli.ts

```diff
@@ -216,6 +216,12 @@ export default function parseCli(argv?: string | readonly string[]) {
         type: "string",
         default: "https://{{ip}}:8935",
       },
+      "ai-gateway-url": {
+        describe:
+          "base URL of the AI Gateway to call for generative AI requests",
+        type: "string",
+        default: "http://staging-livepeer-ai-gateway/", // TODO: Remove this default, staging testing only
```

Comment:
1. What's this TODO saying? Why don't we remove this default right now if we don't want to have it?
2. Do I understand correctly that currently this feature won't work with Livepeer in Box (and it won't be simple to implement it in the box, because Livepeer AI requires GPUs)?

Reply:
This is only from the commit https://github.com/livepeer/studio/pull/2246/commits/802d297f060db3713392c72496787059c8a541ae which will be reverted before merging this. It was used for testing this branch in staging without needing to modify the deployment envs on livepeer-infra

Reply:
> Do I understand correctly that currently this feature won't work with Livepeer in Box (and it won't be simple to implement it in the box, because Livepeer AI requires GPUs)?

Yeah that's correct, so on the box one will just get that error that the AI Gateway is not configured. We need to figure out how to add it E2E in the box, but requiring a 16GB NVIDIA GPU is probably not an option and CPU inference is too slow to be minimally useful.

Maybe the best bet would be requiring you to configure a wallet and run an AI Gateway connected to mainnet, but we can figure that out later especially if/when we get some AI on the video pipeline critical path.

Reply:
Yeah, we can wait with the box implementation.

In general:
- If we believe the box will be used not only for dev, then we can have a URL to AI Gateway
- If we believe box is just for dev env, then we can create an AI Gateway stub